docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Troubleshooting.md

```diff
@@ -58,7 +58,18 @@ Active = 'Y'
 Pending_approval = 'N'
 Password_expiry column value is later than today's date
 ```
-
+If your admin user has been deactivated due to inactivity, you can get past that using the following steps:
+
+1. Log into the to mysql backend with your admin credential
+2.  Make sure the `Active` column in the `users` MySQL table is set to `Y` for this user, if not you can do so with the following command:
+       ```
+       UPDATE users SET active = "Y" WHERE UserID = "admin";
```

Comment:
Just noticed: we use single tick in mysql instead of double quotes, please accept this suggestion @satvik-tha-god --
```suggestion
       UPDATE users SET active = 'Y' WHERE UserID = 'admin';
```